app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str):
+    old_file = command.split()[1]
```

Comment:
it's not old_file rather source file name, rename to source_filename

Reply:
Technically, file 1 is older than file 2. When file 1 already exists, file 2 does not exist. I like variable names like this

Reply:
I agree with Yurii, `old_file` and `new_file` aren't really meaningful